po/fr.po

```diff
@@ -26909,847 +26871,947 @@ msgid "set the list of administrators for GROUP"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:174
 msgid "Skip the signature verification step"
-msgstr ""
+msgstr "Passer la vérification de la signature"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:175
 msgid "Print key listings delimited by colons"
-msgstr ""
+msgstr "Afficher les listes de clés en les délimitant par des doubles points"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:176
 msgid ""
 "Print key listings delimited by colons (like --with-colons) and print the "
 "public key data"
 msgstr ""
+"Afficher les listes de clés en les délimitant par des doubles points (comme "
+"avec --with-colons) et afficher les données de clé publique"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:177
+#, fuzzy
 msgid ""
 "Same as the command --fingerprint but changes only the format of the output "
 "and may be used together with another command"
 msgstr ""
+"Pareil que la commande --fingerprint mais ne changer que le format de "
+"sortie ; peut être utilisé avec une autre commande"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:178
 msgid "Changes the output of the list commands to work faster"
-msgstr ""
+msgstr "Change la sortie des commandes de listage pour travailler plus vite"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:179
 msgid ""
 "Do not merge primary user ID and primary key in --with-colon listing mode "
 "and print all timestamps as UNIX timestamps"
 msgstr ""
+"Ne pas fusionner l’identifiant utilisateur primaire et la clé primaire dans "
+"les listes avec --with-colon, et afficher tous les horodatages au format UNIX"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:180
 msgid ""
 "Changes the behaviour of some commands. This is like --dry-run but different"
 msgstr ""
+"Change le comportement de certaines commandes ; similaire à, quoique "
+"différent de, --dry-run"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:181
 msgid "Display the session key used for one message"
-msgstr ""
+msgstr "Afficher la clé de session utilisée pour un message"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:182
 msgid "Don't use the public key but the specified session key"
-msgstr ""
+msgstr "Utiliser la clé de session spécifiée au lieu de la clé publique"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:183
 #: /tmp/fish/implicit/share/completions/gpg.fish:185
 msgid "Prompt for an expiration time"
-msgstr ""
+msgstr "Demander une date d’expiration"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:184
 #: /tmp/fish/implicit/share/completions/gpg.fish:186
 msgid "Do not prompt for an expiration time"
-msgstr ""
+msgstr "Ne pas demander une date d’expiration"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:187
 msgid ""
 "Don't look at the key ID as stored in the message but try all secret keys in "
 "turn to find the right decryption key"
 msgstr ""
+"Ne pas rechercher l’identifiant de clé tel qu’indiqué dans le message ; "
```

Comment:
See my comment above about the (again, typographically correct) leading space before the semicolon – for line break reasons, we should maybe better disregard this convention, much as it pains the typographer in me. Note this also applies to French quotes, the infamous _guillemets (« and »)_, where there is at least a precedent in spaceless typesetting set by _Libération_.

Reply:
I follow the (almost-)official guidelines (i.e. those from the Imprimerie nationale) for French typography, which recommends using non-breaking space before » and ;, and after «, but typography is something being lost in French. I corrected that, although I doesn't like the idea that what I did was *a correction*.

Reply:
I agree with you, it’s more of a bastardisation than a correction; as I noted, here and elsewhere, your typesetting is not at fault, at all, but 8 bit codepages have no concept of a non-breaking space and I am unsure if we can asssume fish always runs in terminal supporting Unicode characters (i.e. UTF-8). Anyone with input on this (paging @floam @zanchey @krader1961 @ridiculousfish)?